apps/doji-backend/src/review/review.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { CurrentUser, UserAuthGuard } from '@backend/auth/user.guard'
+import { Review } from '@backend/entities/Review'
+import { ReviewAverageRatingDTO, ReviewCreationRequestDTO } from '@backend/review/review.dto'
+import { ReviewService } from '@backend/review/review.service'
+import { IUserReference } from '@libs/api'
+import { Body, Controller, Get, Param, Post, UseGuards } from '@nestjs/common'
+import { ApiCookieAuth } from '@nestjs/swagger'
+
+@Controller('review')
```

Comment:
มีอีกเรื่องที่ยังคิดไม่ออก เนื่องจากว่า review มันอยู่ภายใต้ session เราควรจะย้ายอันนี้ไปอยู่ใน session หรือเปล่า? e.g. ถ้าเรียก GET `/sessions/:sessionId` ก็ให้มันส่ง review มาด้วยเลย ไม่ต้องแยก endpoint

Reply:
ตอนนี้ Review มันเป็นตัวเก็บ Session แบบ Many to One ถ้าจะทำแบบนั้นน่าจะต้องเก็บแบบ ที่ Session เป็นตัวเก็บ Collection ของ Review ปะ

Reply:
Frontend requested that this be placed in as `Session.reviews` [see snippet](https://github.com/2110336-SoftwareEngineering2-2021-2/sec2_ChocoTech/pull/72/files#diff-fa987ed0bd34357401178528fdaae40a636e701a53a014058214a8a5e104b99eR88)